src/scripts/helpers/swipe-tables.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+  'use strict';
+
+  var tableLeftPos, startSwipingPos, swipeRange;
+    
+  function handleSwipe (target) {
+    var mouseX = window.event.clientX;
```

Comment:
It seems like `target.event` might also provide this object. What is the advantage of using the global `window.event`?

See main review comment for alternate suggestions that may not need this code

Reply:
Not working for me `Uncaught TypeError: Cannot read property 'clientX' of undefined`

Reply:
ah, it's because the function on line 14 does not take an argument. I think something like the following would work:

```
target.onmousemove = function(t) {
  handleSwipe(t)
}

handleSwipe(t) {
  console.log(t.clientX)
}
```

I had assumed that `target` was the 1st argument to the function defined at `target.onmousemove = function ...`